index.html

```diff
@@ -42,14 +42,14 @@ <h4>Videos</h4>
           <h4>Admin Username ▽</h4>
       </div>
       <div class="site-logo">
-        <img class="name-logo" src="images/icons/dave-logo.png" alt="share icon" height="85" width="285">
-        <img src="images/icons/glasses-image-2.png" alt="share icon" height="90" width="180">
+        <img class="name-logo" src="images/icons/dave-logo.png" alt="it's dave logo" height="85" width="285">
+        <img src="images/icons/glasses-image-2.png" alt="glasses logo" height="90" width="180">
       </div>
     </div>
   </nav>
   <h1>Shares</h1>
   <section class="user-card-section">
-    <div class="user-card" >
+    <section class="user-card" >
```

Comment:
I can also make comments on specific parts of the code. This can simple be a comment or it can be turned into a change request review that would block merging in a live code situation. This one is just a simple comment

Reply:
Either one of us can respond to this comment and resolve the conversation if we want

Reply:
Thanks for the feedback/review - I like how you can address specific lines of code, makes it clean. After review, I am going to add <a href> elements for the link and include this change in the PR request for HTML.